src/index.html

```diff
@@ -1,13 +1,392 @@
+                <li class="features__item">Party Mode with 6.0 Units and above</li>
+                <li class="features__item">MESHNET Multi Speaker Network</li>
+              </ul>
+            </div>
+            <div class="features__list-container features__list--two">
+              <p class="features__subtitle">App Features</p>
+              <ul class="features__list ">
+                <li class="features__item">Customize Music Schedule</li>
+                <li class="features__item">Wake Up with Favorite Songs</li>
+                <li class="features__item">Home Detection Auto Wake Up</li>
+                <li class="features__item">Color Wheel</li>
+              </ul>
+            </div>
+            <div class="features__list-container">
+              <p class="features__subtitle">Sound & Music</p>
+              <ul class="features__list">
+                <li class="features__item">Feel-in-chest Base Power</li>
+                <li class="features__item">Lossless Digital Audio Transmission</li>
+                <li class="features__item">Easy & Stable Stereo Pairing</li>
+                <li class="features__item">Crisp and Clear High Frequency Sound</li>
+                <li class="features__item">Streams from Cloud Music and Local Library</li>
+                <li class="features__item">Auto Music from Last Song Stopped</li>
+              </ul>
+            </div>
+            <div class="features__icon-container">
+              <img
+                src="./images/icons/feature-element.png"
+                alt="feature-element"
+                class="features__icon"
+              >
+            </div>
+            <div class="features__image-container">
+              <img
+                src="./images/photos/feature.jpg"
+                alt="feature"
+                class="features__image"
+              >
+            </div>
+          </div>
+        </section>
+        <section class="questions" id="get-in-touch">
+          <div class="questions__container">
+            <div class="questions__title-container">
+              <h2 class="questions__title">DO YOU HAVE ANY QUESTIONS?</h2>
+            </div>
+            <form
+              action="#"
+              class="questions__form"
+              id="myForm"
+            >
+              <label for="input-one"></label>
+              <input
+                id="input-one"
+                type="email"
+                class="questions__form-input"
+                placeholder="Your email"
+                required
+              >
+              <textarea
+                class="questions__area"
+                placeholder="Your message..."
+                required
+              ></textarea>
+              <button type="submit" class="questions__button">Send</button>
+            </form>
+          </div>
+        </section>
+      </main>
+      <footer class="footer" id="footer">
```

Comment:
Add a blank line between the containers
```suggestion
      </main>
      
      <footer class="footer" id="footer">
```